mevislab.github.io/content/tutorials/thirdparty/matplotlib/modulesetup.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Example 1: Setting up the module"
+date: 2023-05-26
+status: "OK"
+draft: false
+weight: 881
+tags: ["Advanced", "Tutorial", "Matplotlib", "Visualization"]
+menu: 
+  main:
+    identifier: "matplotlibexample1"
+    title: "Example 1: Setting up the module"
+    weight: 881
+    parent: "matplotlib"
+---
+
+# Example 1: Setting up the module
+
+To be able to acess the data needed for our grayscale distribution plots, we need a network consisting of a module that imports a DICOM, a module that differentiates between slices and another that ouputs histogram data. 
+
+So open up your MeVisLab workspace and add the modules `LocalImage`, `SubImage` and `Histogram` to it.
+Connect the output of `LocalImage` to the input of `SubImage` and the output of `SubImage` with the input of `Histogram`.
```

Comment:
I would always add a 'the' before the modules

Reply:
No - either "the module `LocalImage`" or "`LocalImage`"

Reply:
I didn't as it would elongate the sentence and doesn't contribute to clarity in that case. 
"the modules ...." describes what they are and in the context of a network I didn't feel like I had to repeat that all over again by saying "of the xyz module with the xyz module and the xyz module" -
Do you want me to do it anyways?